ios/Pods/MobileCenter/MobileCenter-SDK-iOS/MobileCenter.framework/Headers/MSConstants.h

```diff
@@ -0,0 +1,47 @@
+#import <Foundation/Foundation.h>
```

Comment:
I also checked with @BretJohnson and he tells me that if we are using the build services, pods are automatically installed. So I don't have to check it in. However, locally someone may have to run `pod install`, if the Pods folder is not checked in.

If you think there is a different way to do it, I would love to hear that. I am also not a great fan of Cocoapods, and I like how in Android, we just have Gradle do the work for us. I also looked at other libraries that have something similar. For example, Bugsnag seems to [check in](https://github.com/bugsnag/bugsnag-react-native/tree/master/cocoa) the pods in npm. The only worry is - if other modules use any library in the vendor folder, there will be conflicts.